packages/account/src/Styles/account.scss

```diff
@@ -22,7 +22,24 @@ $MIN_HEIGHT_FLOATING: calc(
             padding-left: 4.4rem;
             background-color: var(--general-main-1);
         }
+        &__header {
+            &-account__logout {
+                margin-top: 1rem;
+                align-items: center;
+                justify-content: center;
+                display: flex;
+                border: solid 2px $color-grey-7;
```

Comment:
it's better to use var for colors to cover both light and dark theme. just like line 23.